example-scene/src/cameraRotation.ts

```diff
@@ -0,0 +1,83 @@
+import { Entity, engine, Transform, executeTask, MeshRenderer, RaycastResult, Raycast, RaycastQueryType, Billboard, BillboardMode } from "@dcl/sdk/ecs"
+import { Matrix, Quaternion, Vector3 } from "@dcl/sdk/math"
+import { createAxes } from "./axisGenerator"
+
+export function initCameraRotation(parent: Entity) {
+  let renderNumber = 0
+
+  executeTask(async () => {
+    // this line is purposefully a .get instead of a .getOrNull because it
+    // should FAIL ig the Transform of the CameraEntity is not avaliable at this
+    // moment, consider it an assertion
+    Transform.get(engine.CameraEntity)
+  })
+
+  // this entity will copy the rotation of the camera
+  const rotationCopy = engine.addEntity()
+  Transform.create(rotationCopy, { parent, position: Vector3.create(6, 1, 8), scale: Vector3.create(0.5, 0.5, 0.5) })
+  MeshRenderer.setBox(rotationCopy)
+  const axes1 = createAxes()
+  Transform.create(axes1.container, { parent: rotationCopy, scale: Vector3.create(2, 2, 2) })
+
+  // this entity will have a kernel-space billboard
```

Comment:
_Doubt_
Should we use the "kernel-space" and "user-space" terminology? An alternative could be "renderer-managed" vs "scene-managed". I don't know.

Reply:
I think user-space should be there because it is relatable to an existing concept. and renderer-space may be good for consistency, I'd prevent `kernel` as much as possible